nilearn/regions/signal_extraction.py

```diff
@@ -15,8 +15,144 @@
 from .. import masking
 from ..image import new_img_like
 
+INF = 1e-9
+
+
+def _check_shappe_affine_label_img(labels_img, target_shape, target_affine):
+    """Validate shapes and affines of labels.
+    Parameters
+    ----------
+    labels_img : Niimg-like object
+        See http://nilearn.github.io/manipulating_images/input_output.html
+        regions definition as labels. Encodes the region labels of the signals.
+
+    target_shape : numpy.ndarray
+        Desired shape of label images.
+
+    target_affine : numpy.ndarray
+        Desired affine of label images.
+
+    """
+    if labels_img.shape != target_shape:
+        raise ValueError("label img and imgs shapes must be identical.")
+    if labels_img.affine.shape != target_affine.shape or \
+       abs(labels_img.affine - target_affine).max() > INF:
```

Comment:
```suggestion
    if (labels_img.affine.shape != target_affine.shape
        or abs(labels_img.affine - target_affine).max() > INF):
```

Use of `\` is thoroughly discouraged in idiomatic python; instead, parentheses are used for multi-line statement.